readme.md

```diff
@@ -217,6 +217,7 @@
 - [Q#](https://github.com/ebraminio/awesome-qsharp#readme) - Domain-specific programming language used for expressing quantum algorithms.
 - [Imba](https://github.com/koolamusic/awesome-imba#readme) - Programming language inspired by Ruby and Python and compiles to performant JavaScript.
 - [Vala](https://github.com/desiderantes/awesome-vala#readme) - Programming language designed to take full advantage of the GLib and GNOME ecosystems, while preserving the speed of C code.
+- [V](https://github.com/vlang/awesome-v) - Programming language inspired from C, C++, Go and Python.
```

Comment:
Should be at least "inspired by C, C++, Go and Python" or replaced with "official" desciption: "Simple, fast, safe, compiled language for developing maintainable software."

Reply:
Ok, I will go with the official one.